tests/serialize/data.js

```diff
@@ -99,7 +99,7 @@ var doNext = function (remaining) {
             return
           }
           doc = $rdf.sym($rdf.uri.join(right, base))
-          if (doc.uri.slice(0, 8) !== 'file:///') {
+          if (doc.uri.slice(0, 7) !== 'file://') {
```

Comment:
Am minded to put a test in the NamedNode to reject URIs with more than one colon even though people do it a lot 
Maybe we could make a shim layer on Xhr to map to file:///C/.... for our use internally